tests/unit/array/mixins/test_empty.py

```diff
@@ -1,7 +1,11 @@
+import pytest
+
 from docarray import DocumentArray
+from docarray.array.sqlite import DocumentArraySqlite
 
 
-def test_empty_non_zero():
+@pytest.mark.parametrize('da_cls', [DocumentArray, DocumentArraySqlite])
+def test_empty_non_zero(da_cls):
```

Comment:
although empty is possible to `DocumentArraySqlite` (basicly because we allow config-free construction via temp files), I don't think it should be supported for storage backends in general (and no backend in particular).
If we think about it, where does `DocumentArrayWeaviate.empty(10)` store ? Should docs go to the default `localhost:8080` server configuration ?

Reply:
ref1: https://github.com/RaRe-Technologies/sqlitedict/blob/master/sqlitedict.py
ref2: https://github.com/osoken/sqlitecollections

they are support default connection as temp file, so do we. let's follow this behavior